package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "the-travel-shelf",
+  "version": "1.0.0",
+  "description": "A magazine and index of beautiful global independent bookstores",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RichardLitt/the-travel-shelf.git"
+  },
+  "keywords": ["independent", "bookstores", "index"],
+  "author": "Richard Littauer",
+  "license": "MIT",
+  "contributors": ["Joanna Katsifaraki", "Jose Esplana", "Hanie Jafari, Vernon Lillies" ],
+  "bugs": {
+    "url": "https://github.com/RichardLitt/the-travel-shelf/issues"
+  },
+  "homepage": "https://github.com/RichardLitt/the-travel-shelf#readme",
```

Comment:
No need for `#readme`. 

Learn how to remove this and amend the commit. It's ok to force push to a branch (but _only_ to a branch_).